project/Dependencies.scala

```diff
@@ -82,7 +82,9 @@ object Dependencies {
   val slf4jNop = "org.slf4j" % "slf4j-nop" % "1.7.2"
 
   val utest = "com.lihaoyi" %% "utest" % "0.6.9"
-  val pprint = "com.lihaoyi" %% "pprint" % "0.5.5"
+  val pprint = "com.lihaoyi" %% "pprint" % "0.8.0"
+  val oslib = "com.lihaoyi" %% "os-lib" % "0.8.1"
```

Comment:
`pprint` ids must have when developing, `oslib` reduce boilerplate in written testcases